crates/sui-framework/sources/collectible/collectible.move

```diff
@@ -68,6 +69,8 @@ module sui::collectible {
         otw: OTW, max_supply: Option<u64>, ctx: &mut TxContext
     ): (
         Publisher,
+        nft_safe::TransferPolicy<Collectible<T>>,
```

Comment:
These can be created at a later point using the publisher. do we want to export them in this method?